web/files/referencedby.go

```diff
@@ -83,5 +83,5 @@ func RemoveReferencedHandler(c echo.Context) error {
 		return wrapVfsError(err)
 	}
 
-	return c.NoContent(http.StatusNoContent)
+	return c.NoContent(http.StatusOK)
```

Comment:
> Returning 204 causes Go to produce the following error: "http: request method or response status code does not allow body".

How have you got this error?

204 means that the response has no content, and it is the case, as we use `c.NoContent`. But it doesn't mean that the request must be empty.